### PR TITLE
Feature/2938 extend export job

### PR DESCRIPTION
**Pull request by rowasc**
This pull request makes the following changes:
- Depends on #2937 & #2936  being merged
- Implements #2938 
### WIP 
-  has dependencies with open PRs that need to be merged first ( othert ables/repositories needed) 
Test checklist:
- [ ]

- [ ] I certify that I ran my checklist

Fixes ushahidi/platform# .

Ping @ushahidi/platform

### CODE REVIEW

File: migrations/20180512213459_add_hxl_fields_to_export_job.php [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+                'comment' => 'Will we send the file to HDX?',
+            ])
+            ->addColumn(
+                'hxl_heading_row',
+                'text',
+                [
+                    'null' => true,
+                    'limit' => Phinx\Db\Adapter\MysqlAdapter::TEXT_MEDIUM,
+                    'default' => null,
+                    'comment' => 'The computer row of HXL tags+attributes'
```
Comment (kinstelli) [inline migrations/20180512213459_add_hxl_fields_to_export_job.php]: typo: computed

File: migrations/20180512214017_set_export_job_send_to_browser.php [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+{
+
+    public function up()
+    {
+        $sql = "UPDATE export_job SET send_to_browser=true;";
+        $this->execute($sql);
+    }
+    public function down()
+    {
+        // TODO : I'm not sure if we should bring this back to false since we have no way of knowing which
```
Comment (kinstelli) [inline migrations/20180512214017_set_export_job_send_to_browser.php]: agreed

File: src/App/Validator/ExportJob/Update.php [hunk 3]
```diff
@@ -20,13 +20,101 @@ class Update extends Validator
...
+        }
+        return true;
+    }
+
+    /**
+     * @param $first
+     * @param $second
+     * @return bool
+     */
+    private function isOppositeBool($first, $second)
```
Comment (kinstelli) [inline src/App/Validator/ExportJob/Update.php]: Reading the above, I just assumed `isOppositeBool` was already defined in Validator. 
Future: we might as well share this helper there or in a trait or somewhere.

Comment (rowasc) [reply src/App/Validator/ExportJob/Update.php]: yep absolutely. Adding to my list of things to do after release, I'll create a ticket

File: migrations/20180512213459_add_hxl_fields_to_export_job.php (hunk 1 above)
Comment (rowasc) [reply migrations/20180512213459_add_hxl_fields_to_export_job.php]: fixed

**Pull request merged**